tests/rdma/rdma-test.c

```diff
@@ -952,6 +953,39 @@ static void *test_routine(void *arg) {
     }
     printf("Valkey Over RDMA test thread[%d] GET %d KVs [OK]\n", tid, keys);
 
+    /* # round 5, test REPLICAOF, also run REPLICAOF only once */
```

Comment:
TBH, I don't understand the purpose of this test case. REPLICAOF does not communicate with the client from a child process. The command just tells the server to become a replica of another valkey server. The replication between the valkey servers is done in another connection which is not RDMA.

I think we can skip this test case and functionality in the python script to start a replica.

----

If we really want to test a command that communicates with the RDMA client from a child process, I believe the only one is PSYNC with dual channel replication and diskless replication enabled. It means that the RDMA client pretends to be a Valkey replica and sends the commands a replica would do when it starts replicating. There is no use case for this, except only to try a crash the server. It sends REPLCONF and PSYNC with various arguments.

Unfortunately, the PSYNC and REPLCONF commands are not documented, so if we want to write this test, we need to be smart and figure out exactly what a replica does in this scenario. Maybe these comments are useful, but not enough I think: https://github.com/valkey-io/valkey/blob/8.0.1/src/replication.c#L3286

So, it's not very easy to test, and with the protection that prevents the server crash, it will be no crash if a client tries to do this. Therefore, I think we don't have to do this test now. (Maybe in the future.)

Reply:
Sure, remove `REPLICAOF` in the new version. Thanks!